src/test/java/us/abstracta/wiresham/VirtualTcpServiceTest.java

```diff
@@ -115,6 +116,26 @@ public void shouldGetExpectedResponseWhenConnectSsl() throws Exception {
     waitReceived("Hello");
   }
 
+  @Test
+  public void shouldReadWiresharkJsonAndParseServerPort() throws Exception {
+    Flow flow = Flow.fromWiresharkJsonDump(new File(getResourceFilePath("/serverOnLocalPort3469.json")),
+            "127.0.0.1:3469");
+    assertTrue(flow.getSteps().get(0) instanceof ClientPacketStep, "First flow element should be from client");
+    assertTrue(flow.getSteps().get(1) instanceof ServerPacketStep, "Second flow element should be from server");
+    assertTrue(flow.getSteps().get(2) instanceof ClientPacketStep, "Third flow element should be from client");
+    assertTrue(flow.getSteps().get(3) instanceof ServerPacketStep, "Fourth flow element should be from server");
+  }
+
+  @Test
```

Comment:
Consider in this case also renaming test to be more explicit. Maybe something like `shouldGetAllPacketsAsServerWhenLoadWithSameIpForServerAndClientAndPortIsNotSpecified`